### DISCUSSION TITLE
How to change the size of inline preview?

### DISCUSSION THREAD

**Discussion by Flashwalker** [facelessuser/ColorHelper#242] [Q&A] [answered]
How to change the size of inline preview?
And border color?

**Comment by facelessuser** [comment, marked as answer]
You should find the relevant settings here: https://facelessuser.github.io/ColorHelper/settings/graphics/.

**Reply by Flashwalker** [reply]
there is only the `inline_preview_offset` for inline preview,
but nothing about the size

**Reply by facelessuser** [reply]
The literal first setting is called [`graphic_size`](https://facelessuser.github.io/ColorHelper/settings/graphics/#graphic_size).

**Reply by facelessuser** [reply]
Okay, that's probably not a fair statement as that mainly controls popup graphics. Inline previews are sized to line height.  Are you trying to go bigger or smaller?

**Reply by facelessuser** [reply]
`inline_preview_offset` will get you what you want as it offsets the size calculation. Previews are supposed to match line height. When you increase the font size, the previews get bigger as well, but if your system renders them the wrong size, or if you want to just tweak the size, this is the way to do it. To go bigger, make the offset bigger. To go smaller, make the offset smaller. I think we cap the minimum size at some point. It becomes impossible to render a preview properly once we get too small.

**Reply by Flashwalker** [reply]
> Okay, that's probably not a fair statement as that mainly controls popup graphics. Inline previews are sized to line height. Are you trying to go bigger or smaller?

i wana smaller 🙂

**Reply by Flashwalker** [reply]
> I think we cap the minimum size at some point. It becomes impossible to render a preview properly once we get too small.

For some reason I thought `inline_preview_offset` is for the shift, not the size.

Yeap, they don't get smaller than line height

**Reply by facelessuser** [reply]
Yes they do, just use negative numbers as documented:

<img width="240" alt="Screenshot 2023-01-30 at 11 56 45 PM" src="https://user-images.githubusercontent.com/1055125/215688624-bf85259c-d577-481f-9268-31ca6843d631.png">

**Reply by Flashwalker** [reply]
> Yes they do, just use negative numbers as documented:
> 
> <img alt="Screenshot 2023-01-30 at 11 56 45 PM" width="240" src="https://user-images.githubusercontent.com/1055125/215688624-bf85259c-d577-481f-9268-31ca6843d631.png">

Oh, it is
Firstly i tried -2, and didn't notice the difference
Now i see it was too small number
Actualy firstly i thought it should be something like 0.2
